src/controller/CommissioningDelegate.h

```diff
@@ -769,7 +781,9 @@ class CommissioningDelegate
      * kThreadNetworkSetup: NetworkCommissioningStatusInfo if there is an error
      * kWiFiNetworkEnable: NetworkCommissioningStatusInfo if there is an error
      * kThreadNetworkEnable: NetworkCommissioningStatusInfo if there is an error
-     * kFindOperational: OperationalNodeFoundData
+     * kFindOperationalForStayActive OperationalNodeFoundData
```

Comment:
Does this need an entry for kEvictPreviousCaseSessions?

Reply:
updated, thanks